<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Manish Datt">

<title>Enzymes – Slides from bioinfo.guru</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-991b1886d3c685c7aa2b62b80640c7af.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="fullcontent quarto-light">

<div id="quarto-search-results"></div>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Enzymes</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Manish Datt </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<ul>
<li>Protein structure</li>
<li>Protein function</li>
</ul>
</section>
<section id="enzymes" class="level2">
<h2 class="anchored" data-anchor-id="enzymes">Enzymes</h2>
<ul>
<li>Catalysis</li>
<li>Proteins</li>
</ul>
<hr>
<section id="quantitative-description-of-protein-ligand-interactions" class="level3">
<h3 class="anchored" data-anchor-id="quantitative-description-of-protein-ligand-interactions">Quantitative description of protein-ligand interactions</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Function in biochemistry often revolves around a reversible protein-ligand interaction. <span class="math display">\[\begin{align}
P + L \rightleftharpoons PL
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
K_a = \frac{[PL]}{[P][L]} \\
\end{align}\]</span> A higher value of <span class="math inline">\(K_a\)</span> corresponds to a higher affinity of the ligand for the protein.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
K_a[L] = \frac{[PL]}{[P]}
\end{align}\]</span> The ratio of bound to free protein is directly proportional to the concentration of free ligand.</p>
</div>
</div>
<hr>
</section>
<section id="occupancy-of-ligand-binding-sites" class="level3">
<h3 class="anchored" data-anchor-id="occupancy-of-ligand-binding-sites">Occupancy of ligand-binding sites</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\theta = \frac{\text{Binding sites occupied}}{\text{Total binding sites}} = \frac{[PL]}{[PL] + [P]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Substituting <span class="math inline">\(K_a[L][P]\)</span> for <span class="math inline">\([PL]\)</span> <span class="math display">\[\begin{align}
\theta = \frac{K_a[L][P]}{K_a[L][P] + [P]} = \frac{K_a[L]}{K_a[L] + 1} = \frac{[L]}{[L] + \frac{1}{K_a}}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:60%;">
<p>The value of <span class="math inline">\(K_a\)</span> can be determined from a plot of <span class="math inline">\(\theta\)</span> versus the concentration of free ligand, [L].</p>
<p>The [L] at which half of the available ligand-binding sites are occupied (at <span class="math inline">\(\theta\)</span> = 0.5) corresponds to <span class="math inline">\(\frac{1}{K_a}\)</span>.</p>
</div><div class="column" style="width:40%;">
<p><img src="images/theta_ligand.png" class="img-fluid" width="400"></p>
</div>
</div>
</div>
</div>
<hr>
</section>
<section id="dissociation-constant" class="level3">
<h3 class="anchored" data-anchor-id="dissociation-constant">Dissociation constant</h3>
<p><span class="math inline">\(K_d\)</span> is the equilibrium constant for the release of ligand.</p>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
K_d = \frac{[P][L]}{[PL]} \\
[PL] = \frac{[P][L]}{K_d} \\
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\theta = \frac{[L]}{[L] + K_d}
\end{align}\]</span> When <span class="math inline">\([L]\)</span> is equal to <span class="math inline">\(K_d\)</span>, half of the ligand-binding sites are occupied.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>As <span class="math inline">\([L]\)</span> falls below <span class="math inline">\(K_d\)</span>, progressively less of the protein has ligand bound to it.</p>
<p>In order for 90% of the available ligand-binding sites to be occupied, <span class="math inline">\([L]\)</span> must be nine times greater than <span class="math inline">\(K_d\)</span>.</p>
</div>
</div>
<hr>
</section>
<section id="dissociation-constant-1" class="level3">
<h3 class="anchored" data-anchor-id="dissociation-constant-1">Dissociation constant</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math inline">\(K_d\)</span> is used much more often than <span class="math inline">\(K_a\)</span> to express the affinity of a protein for a ligand.</p>
<p>A lower value of <span class="math inline">\(K_d\)</span> corresponds to a higher affinity of the ligand for the protein.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math inline">\(K_d\)</span> is equivalent to the molar concentration of ligand at which half of the available ligand-binding sites are occupied.</p>
<p>At this point, the protein is said to have reached half-saturation with respect to ligand binding.</p>
</div>
</div>
<div class="fragment fade-in">
<p>The more tightly a protein binds a ligand, the lower the concentration of ligand required for half the binding sites to be occupied, and thus the lower the value of <span class="math inline">\(K_d\)</span>.</p>
</div>
<hr>
</section>
<section id="protein-structure-affects-ligand-binding" class="level3">
<h3 class="anchored" data-anchor-id="protein-structure-affects-ligand-binding">Protein structure affects ligand binding</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>The interaction is greatly affected by protein structure and is often accompanied by conformational changes.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Carbon monoxide binds to free heme molecules more than 20,000 times better than does O<sub>2</sub> (that is, the <span class="math inline">\(K_d\)</span> or P<sub>50</sub> for CO binding to free heme is more than 20,000 times lower than that for O<sub>2</sub>).</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:60%;">
<p>CO binds only about 200 times better when the heme is bound in myoglobin.</p>
<div class="fragment fade-in">
<p>The <em>distal His</em> (His<sup>64</sup> E7), does not affect the binding of O<sub>2</sub> but may preclude the linear binding of CO.</p>
</div>
<div class="fragment fade-in">
<p>A reduction in CO binding is physiologically important, because CO is a low-level byproduct of cellular metabolism.</p>
</div>
</div><div class="column" style="width:40%;">
<p><img src="images/Heme_O2_CO.png" class="img-fluid" width="325"></p>
</div>
</div>
</div>
</div>
<hr>
</section>
<section id="enzymes---introduction" class="level3">
<h3 class="anchored" data-anchor-id="enzymes---introduction">Enzymes - Introduction</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Much of the history of biochemistry is the history of enzyme research.</p>
<p>Biological catalysis was first recognized and described in the late 1700s,</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>In 1897, Eduard Buchner discovered that yeast extracts could ferment sugar to alcohol, proving that fermentation was promoted by molecules that continued to function when removed from cells.</p>
</div>
</div>
<div class="notes">
<p>German physiologist FWK - ancient greek.</p>
<p>As vitalistic notions of life were disproved, the isolation of new enzymes and the investigation of their properties advanced the science of biochemistry.</p>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Frederick W. Kühne called these molecules <strong>enzymes</strong>.</p>
<p>ἐν (en, “in”) +‎ ζύμη (zúmē, “sourdough”)</p>
</div>
</div>
<div class="fragment fade-in">
<div style="margin-top:1em; background-color: lightblue; padding:0.5em">
<p>A substance produced by a living organism which acts as a catalyst to bring about a specific biochemical reaction.</p>
</div>
</div>
<hr>
</section>
<section id="most-enzymes-are-proteins" class="level3">
<h3 class="anchored" data-anchor-id="most-enzymes-are-proteins">Most enzymes are proteins</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>James Sumner found that urease crystals consisted entirely of protein, and he postulated that all enzymes are proteins.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>In 1930, J. B. S. Haldane made the remarkable suggestion that weak bonding interactions between an enzyme and its substrate might be used to catalyze a reaction.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>With the exception of a small group of catalytic RNA molecules, all enzymes are proteins.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Their catalytic activity depends on the integrity of their native protein conformation.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:70%;">
<div style="margin-right:-0.8em;">
<p>Some enzymes require no chemical groups for activity other than their amino acid residues.</p>
<div class="fragment fade-in">
<p>An additional chemical component called a <strong>cofactor</strong>—either one or more inorganic ions or a complex organic or metalloorganic molecule called a <strong>coenzyme</strong>.</p>
</div>
</div>
</div><div class="column" style="width:30%;">
<p><img src="images/cofactors.png" class="img-fluid" width="325"></p>
</div>
</div>
</div>
</div>
<hr>
</section>
<section id="cofactors-and-coenzymes" class="level3">
<h3 class="anchored" data-anchor-id="cofactors-and-coenzymes">Cofactors and coenzymes</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Some enzymes require both a coenzyme and a cofactor for activity.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>A coenzyme or metal ion that is very tightly or even covalently bound to the enzyme protein is called a <strong>prosthetic group</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>A complete, catalytically active enzyme together with its bound coenzyme and/or metal ions is called a <strong>holoenzyme</strong>.</p>
<p>The protein part of such an enzyme is called the <strong>apoenzyme</strong> or <strong>apoprotein</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Coenzymes act as transient carriers of specific functional groups.</p>
<p>Most are derived from vitamins, organic nutrients required in small amounts in the diet.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Some enzymes are modified covalently by phosphorylation, glycosylation, and other processes.</p>
<p>Many of these alterations are involved in the regulation of enzyme activity.</p>
</div>
</div>
<hr>
</section>
<section id="enzyme-classification" class="level3">
<h3 class="anchored" data-anchor-id="enzyme-classification">Enzyme Classification</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Many enzymes have been named by adding the suffix “-ase” to the name of their substrate or to a word or phrase describing their activity.</p>
</div>
</div>
<div class="notes">
<p>Thus urease catalyzes hydrolysis of urea, and DNA polymerase catalyzes the polymerization of nucleotides to form DNA.</p>
<p>An enzyme known to act in the digestion of foods was named pepsin, from the Greek pepsis, “digestion,” and lysozyme was named for its ability to lyse bacterial cell walls.</p>
<p>Transfer of a phosphoryl group from ATP to glucose.</p>
<p>Number 2 denote class name (transferase); the second number (7), the subclass (phosphotransferase); the third number (1), a phosphotransferase with a hydroxyl group as acceptor; and the fourth number (1), D-glucose as the phosphoryl group acceptor. For many enzymes, a trivial name is more commonly used—in this case hexokinase.</p>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><img src="images/Enzyme_class.png" class="img-fluid"></p>
<p>Each enzyme is assigned a four-part classification number and a systematic name, which identifies the reaction it catalyzes.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\text{ATP} + \text{D-glucose} \rightarrow \text{ADP} + \text{D-glucose-6-phosphate}
\end{align}\]</span></p>
<p>Enzyme name: <strong>ATP:glucose phosphotransferase</strong> &nbsp;&nbsp;&nbsp;&nbsp; E.C. number <strong>2.7.1.1</strong></p>
</div>
</div>
<hr>
</section>
<section id="how-enzymes-work" class="level3">
<h3 class="anchored" data-anchor-id="how-enzymes-work">How enzymes work</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Under biologically relevant conditions, uncatalyzed reactions tend to be slow—most biological molecules are quite stable in the neutral-pH, mild temperature, aqueous environment inside cells.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Many common reactions in biochemistry entail chemical events that are unfavorable or unlikely in the cellular environment, such as the transient formation of unstable charged intermediates or the collision of two or more molecules in the precise orientation required for reaction.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>An enzyme circumvents these problems by providing a specific environment within which a given reaction can occur more rapidly.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>The distinguishing feature of an enzyme-catalyzed reaction is that it takes place within the confines of a pocket on the enzyme called the <strong>active site</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Often, the active site encloses a <strong>substrate</strong>, sequestering it completely from solution.</p>
</div>
</div>
<hr>
</section>
<section id="enzyme-action" class="level3">
<h3 class="anchored" data-anchor-id="enzyme-action">Enzyme action</h3>
<div style="font-size: 0.8em; margin-top:-1em;">
<p><span class="math display">\[\begin{align}
E + S \rightleftharpoons ES \rightleftharpoons EP \rightleftharpoons E + P
\end{align}\]</span></p>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em; margin-top:-0.5em;">
<p>The function of a catalyst is to increase the <strong>rate</strong> of a reaction. Catalysts do not affect reaction <strong>equilibria</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em; margin-top:-0.5em;">
<p>The free energy of the ground state of P is lower than that of S, so <span class="math inline">\(\Delta\text{G}^{′°}\)</span> for the reaction is negative and the equilibrium favors P.</p>
<p>A favorable equilibrium does not mean that the S <span class="math inline">\(\rightarrow\)</span> P conversion will occur at a detectable rate (energy “hill” ).</p>
</div>
</div><div class="column" style="width:40%;">
<p><img src="images/Enzyme_RS1.png" class="img-fluid"></p>
</div>
</div>
</div>
<div class="fragment fade-in">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em; margin-top:-0.5em;">
<p>The difference between the energy levels of the ground state and the transition state is the <strong>activation energy, <span class="math inline">\(\Delta\text{G}\)</span><sup>‡</sup></strong>.</p>
<p>Catalysts enhance reaction rates by lowering activation energies.</p>
</div>
</div><div class="column" style="width:40%;">
<div style="margin-top:-1em;">
<p><img src="images/Enzyme_RS2.png" class="img-fluid"></p>
</div>
</div>
</div>
</div>
<hr>
<section id="substrate-concentration-affects-the-rate-of-enzyme-catalyzed-reactions" class="level5">
<h5 class="anchored" data-anchor-id="substrate-concentration-affects-the-rate-of-enzyme-catalyzed-reactions">Substrate Concentration Affects the Rate of Enzyme-Catalyzed Reactions</h5>
<div class="fragment">
<div class="columns">
<div class="column" style="width:40%;">
<p><img src="images/Vmax.png" class="img-fluid"></p>
</div><div class="column" style="width:60%;">
<div style="font-size: 0.8em; margin-top:-1em;">
<p>Initial rate (<strong>V<sub>0</sub></strong>), when [S] &gt;&gt; [E].</p>
<p><span class="math display">\[\begin{align}
E + S \rightleftharpoons ES \;\;\;\;\;\; \text{(fast)}\\
ES \rightleftharpoons E + P \;\;\;\;\;\; \text{(slow)}
\end{align}\]</span></p>
<div class="fragment fade-in">
<p>The reaction quickly achieves a steady state in which [ES] remains approximately constant over time (<strong>steady-state kinetics</strong>).</p>
</div>
</div>
</div>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
E + S \underset{k_{-1}}{\stackrel{k_1}{\rightleftharpoons}} ES \underset{}{\stackrel{k2}{\rightarrow}} E + P
\end{align}\]</span></p>
</div>
</div>
<div class="notes">
<p>Assumption, P to S is ignored i.e.&nbsp;no k-2.</p>
<p>V0 is determined by the breakdown of ES to form product, which is determined by [ES]</p>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
V_o = k_2 [ES]
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>[ES] can not be easily measured experimentally.</p>
</div>
</div>
<hr>
</section>
</section>
<section id="enzyme-kinetics" class="level3">
<h3 class="anchored" data-anchor-id="enzyme-kinetics">Enzyme kinetics</h3>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\text{Rate of ES formation = } k_1 ([E_t] - [ES])[S] \\
\text{Rate of ES breakdown = } k_{-1} [ES] + k_2 [ES]
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><strong>Steady-state assumption:</strong> Rate of formation of ES is equal to the rate of its breakdown. <span class="math display">\[\begin{align}
k_1 ([E_t] - [ES])[S] =  k_{-1} [ES] + k_2 [ES]
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
[ES] =  \frac{[E_t][S]}{[S] + (k_2 + k_{-1})/k_1}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
[ES] =  \frac{[E_t][S]}{K_m + [S]}
\end{align}\]</span> <span class="math inline">\(K_m\)</span> is <strong>Michaelis constant</strong>.</p>
</div>
</div>
<hr>
</section>
<section id="michaelis-menten-equation" class="level3">
<h3 class="anchored" data-anchor-id="michaelis-menten-equation">Michaelis-Menten Equation</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Expressing <span class="math inline">\(V_0\)</span> in terms of <span class="math inline">\([ES]\)</span>. <span class="math display">\[\begin{align}
V_0 =  \frac{k_2 [E_t][S]}{K_m + [S]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Maximum velocity occurs when the enzyme is saturated (i.e.&nbsp;<span class="math inline">\([ES] = [E_t]\)</span>).</p>
<div style="font-size: 1em;">
<p><span class="math display">\[\begin{align}
V_0 =  \frac{V_{max} [S]}{K_m + [S]}
\end{align}\]</span></p>
</div>
</div>
</div>
<div class="notes">
<p>This is the Michaelis-Menten equation, the rate equation for a one-substrate enzyme-catalyzed reaction. It is a statement of the quantitative relationship between the initial velocity V0, the maximum velocity Vmax, and the initial substrate concentration [S], all related through the Michaelis constant Km. Note that Km has units of concentration. Does the equation fit experimental observations? Yes; we can confirm this by considering the limiting situations where [S] is very high or very low, as shown in Figure 6–12.</p>
<p>This is a very useful, practical definition of Km: Km is equivalent to the substrate concentration at which V0 is one-half Vmax.</p>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>When <span class="math inline">\(V_o\)</span> is half <span class="math inline">\(V_{max}\)</span> <span class="math display">\[\begin{align}
\frac{V_{max}}{2} =  \frac{V_{max} [S]}{K_m + [S]} =&gt; \frac{1}{2} = \frac{[S]}{K_m + [S]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Solving of <span class="math inline">\(K_m\)</span>, we get <span class="math inline">\(K_m + [S] = 2[S]\)</span> or <span class="math inline">\(K_m = [S]\)</span>, when <span class="math inline">\(V_0 = \frac{1}{2}V_{max}\)</span></p>
</div>
</div>
<hr>
</section>
<section id="double-reciprocal-plot" class="level3">
<h3 class="anchored" data-anchor-id="double-reciprocal-plot">Double-Reciprocal Plot</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
V_0 =  \frac{V_{max} [S]}{K_m + [S]} \text{ Reciprocal =&gt; }  \frac{1}{V_0} = \frac{K_m + [S]}{V_{max}[S]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\frac{1}{V_0} = \frac{K_m}{V_{max}[S]} + \frac{[S]}{V_{max}[S]} =&gt;
\frac{1}{V_0} = \frac{K_m}{V_{max}[S]} + \frac{1}{V_{max}}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em;">
<p>A double-reciprocal plot or <strong>Lineweaver-Burk plot</strong>.</p>
<div class="fragment fade-in">
<p>Slope =&gt; <span class="math inline">\(\frac{K_m}{V_{max}}\)</span> Y intercept =&gt; <span class="math inline">\(\frac{1}{V_{max}}\)</span> &nbsp;&nbsp; X intercept =&gt; <span class="math inline">\(-\frac{1}{K_m}\)</span></p>
</div>
<div class="fragment fade-in">
<p>The double-reciprocal plot of enzyme reaction rates is very useful in distinguishing between certain types of enzymatic reaction mechanisms and in analyzing enzyme inhibition.</p>
</div>
</div>
</div><div class="column" style="width:40%;">
<div style="font-size: 0.8em;">
<p><img src="images/Double_reciprocal.png" class="img-fluid" width="400"></p>
</div>
</div>
</div>
</div>
</div>
<hr>
</section>
<section id="turnover-number" class="level3">
<h3 class="anchored" data-anchor-id="turnover-number">Turnover number</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Number of substrate molecules converted to product in a given unit of time on a single enzyme molecule when the enzyme is saturated with substrate.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Each enzyme has values of <span class="math inline">\(k_{cat}\)</span> and <span class="math inline">\(K_m\)</span> that reflect the cellular environment, the concentration of substrate normally encountered in vivo by the enzyme, and the chemistry of the reaction being catalyzed.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Two enzymes catalyzing different reactions may have the same <span class="math inline">\(k_{cat}\)</span> (turnover number), yet the rates of the uncatalyzed reactions may be different and thus the rate enhancements brought about by the enzymes may differ greatly.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>The best way to compare the catalytic efficiencies of different enzymes or the turnover of different substrates by the same enzyme is to compare the ratio <span class="math inline">\(\frac{k_{cat}}{K_m}\)</span> for the two reactions.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math inline">\(\frac{k_{cat}}{K_m}\)</span> is called the <strong>specificity constant</strong>.</p>
</div>
</div>
<div class="notes">
<p>This parameter, sometimes called the specificity constant, is the rate constant for the conversion of E+S to E+P. When [S] &lt;&lt; Km.</p>
<p>V0 in this case depends on the concentration of two reactants, [Et] and [S]; therefore this is a second-order rate equation and the constant.</p>
<p>There is an upper limit to kcat/Km, imposed by the rate at which E and S can diffuse together in an aqueous solution. This diffusion controlled limit is 10e8 to 10e9 M-1 s-1</p>
<p>Many enzymes have a kcat/Km near this range (Table 6–8). Such enzymes are said to have achieved catalytic perfection. Note that different values of kcat and Km can produce the maximum ratio</p>
</div>
<hr>
</section>
<section id="presteady-state-kinetics" class="level3">
<h3 class="anchored" data-anchor-id="presteady-state-kinetics">Pre–Steady State Kinetics</h3>
<div class="fragment">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em;">
<p>Sequential vs double-displacement (ping-pong) reactions.</p>
<div class="fragment fade-in">
<p>In Sequential kinetics there is formation of a ternary complex.</p>
</div>
<div class="fragment fade-in">
<p>In ping-pong kinetics one of the products is release before all the substrates have bound to the enzyme.</p>
</div>
<div class="fragment fade-in">
<div style="margin-top: -1em;">
<p><img src="images/ping_pong.png" class="img-fluid" width="500"></p>
</div>
</div>
</div>
</div><div class="column" style="width:40%;">
<div style="font-size: 0.8em;">
<p><img src="images/Double_reciprocal_2.png" class="img-fluid" width="400"></p>
</div>
</div>
</div>
</div>
<div class="notes">
<p>The sequential mechanism can be ordered or random (for substrate binding).</p>
<p>Kinetic Tests for Determining Inhibition Mechanisms</p>
</div>
<hr>
</section>
<section id="regulatory-enzymes" class="level3 smaller">
<h3 class="smaller anchored" data-anchor-id="regulatory-enzymes">Regulatory Enzymes</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Each pathway, however, includes one or more enzymes that have a greater effect on the rate of the overall sequence. These regulatory enzymes exhibit increased or decreased catalytic activity in response to certain signals.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Mostly, the first enzyme of the sequence is a regulatory enzyme.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><strong>Allosteric enzymes</strong> function through reversible, noncovalent binding of regulatory compounds called allosteric modulators or allosteric effectors (small metabolites or cofactors).</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Other enzymes are regulated by reversible <strong>covalent modification</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Both classes of regulatory enzymes tend to be multisubunit proteins, and in some cases the regulatory site(s) and the active site are on separate subunits.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Some enzymes are stimulated or inhibited when they are bound by separate regulatory proteins.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Others are activated when peptide segments are removed by proteolytic cleavage.</p>
</div>
</div>
<div class="notes">
<p>Examples of enzymes activated by proteolytic cleavage? What is feedback inhibition?</p>
</div>
<hr>
</section>
<section id="allosteric-enzymes" class="level3">
<h3 class="anchored" data-anchor-id="allosteric-enzymes">Allosteric enzymes</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><img src="images/Allostery.png" class="img-fluid"></p>
</div>
</div>
<div class="notes">
<p>The Kinetic Properties of Allosteric Enzymes Diverge from Michaelis-Menten Behavior</p>
<p>Examples of allosteric enzymes? Aspartate transcarbomylase</p>
<p>Enzymes at important metabolic intersections may be regulated by complex combinations of effectors, allowing coordination of the activities of interconnected pathways.</p>
</div>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>