<!DOCTYPE html>
<html lang="en"><head>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-html/tabby.min.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/quarto-contrib/roughnotation-0.5.1/rough-notation.iife.js"></script>
<script src="../site_libs/quarto-contrib/roughnotation-init-1.0.0/rough.js"></script><meta charset="utf-8">
  <meta name="generator" content="quarto-1.7.32">

  <meta name="author" content="Manish Datt">
  <title>TCA</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
    /* CSS for syntax highlighting */
    html { -webkit-text-size-adjust: 100%; }
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; font-weight: bold; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
  </style>
  <link rel="stylesheet" href="../site_libs/revealjs/dist/theme/quarto-f563837468303362081e247dddd440d0.css">
  <link href="../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">TCA</h1>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Manish Datt 
</div>
</div>
</div>

</section>
<section id="introduction" class="slide level2">
<h2>Introduction</h2>
<ul>
<li>Tricarboxylic Acid Cycle</li>
<li>Mitochondria</li>
</ul>
</section>
<section id="tca" class="slide level2">
<h2>TCA</h2>
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class=""><p></p>
<div>
<svg width="960" height="480" viewbox="0.00 0.00 842.84 651.32" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" style="; max-width: none; max-height: none; display: block; margin: auto auto auto auto">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 647.32)">
<title>TCA</title>
<polygon fill="white" stroke="transparent" points="-4,4 -4,-647.32 838.84,-647.32 838.84,4 -4,4"></polygon>
<!-- node1 -->
<g id="node1" class="node">
<title>node1</title>
<ellipse fill="none" stroke="black" cx="701.27" cy="-123.46" rx="37.13" ry="18"></ellipse>
<text text-anchor="middle" x="701.27" y="-119.26" font-family="Times,serif" font-size="14.00">Citrate</text>
</g>
<!-- node2 -->
<g id="node2" class="node">
<title>node2</title>
<ellipse fill="none" stroke="black" cx="518.61" cy="-18" rx="63.69" ry="18"></ellipse>
<text text-anchor="middle" x="518.61" y="-13.8" font-family="Times,serif" font-size="14.00">cis-Aconitase</text>
</g>
<!-- node1&#45;&gt;node2 -->
<g id="edge1" class="edge">
<title>node1-&gt;node2</title>
<path fill="none" stroke="black" d="M677.01,-109.46C645.97,-91.53 591.88,-60.31 555.66,-39.39"></path>
<polygon fill="black" stroke="black" points="557.28,-36.28 546.87,-34.32 553.78,-42.35 557.28,-36.28"></polygon>
<text text-anchor="middle" x="588.35" y="-78.63" font-family="Times,serif" font-size="14.00">Aconitase</text>
</g>
<!-- node3 -->
<g id="node3" class="node">
<title>node3</title>
<ellipse fill="none" stroke="black" cx="310.89" cy="-54.63" rx="46.92" ry="18"></ellipse>
<text text-anchor="middle" x="310.89" y="-50.43" font-family="Times,serif" font-size="14.00">Isocitrate</text>
</g>
<!-- node2&#45;&gt;node3 -->
<g id="edge2" class="edge">
<title>node2-&gt;node3</title>
<path fill="none" stroke="black" d="M464.58,-27.53C433.58,-32.99 394.83,-39.83 363.99,-45.26"></path>
<polygon fill="black" stroke="black" points="363.1,-41.87 353.86,-47.05 364.31,-48.76 363.1,-41.87"></polygon>
<text text-anchor="middle" x="386.29" y="-40.6" font-family="Times,serif" font-size="14.00">Aconitase</text>
</g>
<!-- node4 -->
<g id="node4" class="node">
<title>node4</title>
<ellipse fill="none" stroke="black" cx="175.31" cy="-216.2" rx="71.77" ry="18"></ellipse>
<text text-anchor="middle" x="175.31" y="-212" font-family="Times,serif" font-size="14.00">α-Ketoglutarate</text>
</g>
<!-- node3&#45;&gt;node4 -->
<g id="edge3" class="edge">
<title>node3-&gt;node4</title>
<path fill="none" stroke="black" d="M296.38,-71.92C272.45,-100.44 224.59,-157.47 196.75,-190.65"></path>
<polygon fill="black" stroke="black" points="193.88,-188.63 190.13,-198.54 199.24,-193.13 193.88,-188.63"></polygon>
<text text-anchor="middle" x="176.79" y="-135.49" font-family="Times,serif" font-size="14.00">Isocitrate dehydrogenase</text>
</g>
<!-- node5 -->
<g id="node5" class="node">
<title>node5</title>
<ellipse fill="none" stroke="black" cx="175.31" cy="-427.12" rx="66.08" ry="18"></ellipse>
<text text-anchor="middle" x="175.31" y="-422.92" font-family="Times,serif" font-size="14.00">Succinyl-CoA</text>
</g>
<!-- node4&#45;&gt;node5 -->
<g id="edge4" class="edge">
<title>node4-&gt;node5</title>
<path fill="none" stroke="black" d="M175.31,-234.55C175.31,-271.49 175.31,-355.05 175.31,-398.96"></path>
<polygon fill="black" stroke="black" points="171.81,-398.97 175.31,-408.97 178.81,-398.97 171.81,-398.97"></polygon>
<text text-anchor="middle" x="87.66" y="-320.96" font-family="Times,serif" font-size="14.00">α-Ketoglutarate dehydrogenase</text>
</g>
<!-- node6 -->
<g id="node6" class="node">
<title>node6</title>
<ellipse fill="none" stroke="black" cx="310.89" cy="-588.7" rx="48.65" ry="18"></ellipse>
<text text-anchor="middle" x="310.89" y="-584.5" font-family="Times,serif" font-size="14.00">Succinate</text>
</g>
<!-- node5&#45;&gt;node6 -->
<g id="edge5" class="edge">
<title>node5-&gt;node6</title>
<path fill="none" stroke="black" d="M190.08,-444.72C214.12,-473.37 261.82,-530.22 289.55,-563.26"></path>
<polygon fill="black" stroke="black" points="287.03,-565.71 296.14,-571.12 292.4,-561.21 287.03,-565.71"></polygon>
<text text-anchor="middle" x="168.86" y="-508.19" font-family="Times,serif" font-size="14.00">Succinyl-CoA synthetase</text>
</g>
<!-- node7 -->
<g id="node7" class="node">
<title>node7</title>
<ellipse fill="none" stroke="black" cx="518.61" cy="-625.32" rx="47.51" ry="18"></ellipse>
<text text-anchor="middle" x="518.61" y="-621.12" font-family="Times,serif" font-size="14.00">Fumarate</text>
</g>
<!-- node6&#45;&gt;node7 -->
<g id="edge6" class="edge">
<title>node6-&gt;node7</title>
<path fill="none" stroke="black" d="M354.94,-596.46C387.1,-602.13 430.87,-609.85 465.06,-615.88"></path>
<polygon fill="black" stroke="black" points="464.88,-619.4 475.33,-617.69 466.09,-612.51 464.88,-619.4"></polygon>
<text text-anchor="middle" x="339.06" y="-610.37" font-family="Times,serif" font-size="14.00">Succinate dehydrogenase</text>
</g>
<!-- node8 -->
<g id="node8" class="node">
<title>node8</title>
<ellipse fill="none" stroke="black" cx="701.27" cy="-519.86" rx="46.92" ry="18"></ellipse>
<text text-anchor="middle" x="701.27" y="-515.66" font-family="Times,serif" font-size="14.00">L-Malate</text>
</g>
<!-- node7&#45;&gt;node8 -->
<g id="edge7" class="edge">
<title>node7-&gt;node8</title>
<path fill="none" stroke="black" d="M544.78,-610.21C576.65,-591.81 630.63,-560.65 666.11,-540.16"></path>
<polygon fill="black" stroke="black" points="668.24,-542.97 675.15,-534.94 664.74,-536.91 668.24,-542.97"></polygon>
<text text-anchor="middle" x="578.23" y="-579.39" font-family="Times,serif" font-size="14.00">Fumerase</text>
</g>
<!-- node9 -->
<g id="node9" class="node">
<title>node9</title>
<ellipse fill="none" stroke="black" cx="773.41" cy="-321.66" rx="61.37" ry="18"></ellipse>
<text text-anchor="middle" x="773.41" y="-317.46" font-family="Times,serif" font-size="14.00">Oxaloacetate</text>
</g>
<!-- node8&#45;&gt;node9 -->
<g id="edge8" class="edge">
<title>node8-&gt;node9</title>
<path fill="none" stroke="black" d="M707.8,-501.92C720.46,-467.15 748.4,-390.38 763.41,-349.14"></path>
<polygon fill="black" stroke="black" points="766.71,-350.3 766.84,-339.7 760.13,-347.9 766.71,-350.3"></polygon>
<text text-anchor="middle" x="672.44" y="-429.73" font-family="Times,serif" font-size="14.00">Malate dehydrogenase</text>
</g>
<!-- node9&#45;&gt;node1 -->
<g id="edge9" class="edge">
<title>node9-&gt;node1</title>
<path fill="none" stroke="black" d="M766.88,-303.72C754.19,-268.86 726.14,-191.79 711.16,-150.62"></path>
<polygon fill="black" stroke="black" points="714.44,-149.42 707.73,-141.22 707.86,-151.81 714.44,-149.42"></polygon>
<text text-anchor="middle" x="694.11" y="-231.37" font-family="Times,serif" font-size="14.00">Citrate synthase</text>
</g>
</g>
</svg>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</section>
<section class="slide level2">

<div class="sourceCode" id="cb1"><pre class="sourceCode numberSource {dot} number-lines code-with-copy"><code class="sourceCode"><span id="cb1-1"><a></a>digraph G {</span>
<span id="cb1-2"><a></a>  layout=circo</span>
<span id="cb1-3"><a></a>  Citrate -&gt; "cis-Aconitate";</span>
<span id="cb1-4"><a></a>  "cis-Aconitate" -&gt; Isocitrate;</span>
<span id="cb1-5"><a></a>  Isocitrate -&gt; "&amp;#945;-Ketoglutarate";</span>
<span id="cb1-6"><a></a>  "&amp;#945;-Ketoglutarate" -&gt; "Succinyl-CoA";</span>
<span id="cb1-7"><a></a>  "Succinyl-CoA" -&gt; Succinate;</span>
<span id="cb1-8"><a></a>  Succinate -&gt; Fumarate;</span>
<span id="cb1-9"><a></a>  Fumarate -&gt; "L-Malate";</span>
<span id="cb1-10"><a></a>  "L-Malate" -&gt; Oxaloacetate;</span>
<span id="cb1-11"><a></a>  Oxaloacetate -&gt; Citrate;</span>
<span id="cb1-12"><a></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</section>
<section class="slide level2">

<h3 id="quantitative-description-of-protein-ligand-interactions">Quantitative description of protein-ligand interactions</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Function in <span class="rn" data-rn-type="circle" data-rn-color="blue" data-rn-animate="true">biochemistry</span> often revolves around a reversible protein-ligand interaction. <span class="math display">\[\begin{align}
P + L \rightleftharpoons PL
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
K_a = \frac{[PL]}{[P][L]} \\
\end{align}\]</span> A higher value of <span class="math inline">\(K_a\)</span> corresponds to a higher affinity of the ligand for the protein.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
K_a[L] = \frac{[PL]}{[P]}
\end{align}\]</span> The ratio of bound to free protein is directly proportional to the concentration of free ligand.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="occupancy-of-ligand-binding-sites">Occupancy of ligand-binding sites</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\theta = \frac{\text{Binding sites occupied}}{\text{Total binding sites}} = \frac{[PL]}{[PL] + [P]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Substituting <span class="math inline">\(K_a[L][P]\)</span> for <span class="math inline">\([PL]\)</span> <span class="math display">\[\begin{align}
\theta = \frac{K_a[L][P]}{K_a[L][P] + [P]} = \frac{K_a[L]}{K_a[L] + 1} = \frac{[L]}{[L] + \frac{1}{K_a}}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:60%;">
<p>The value of <span class="math inline">\(K_a\)</span> can be determined from a plot of <span class="math inline">\(\theta\)</span> versus the concentration of free ligand, [L].</p>
<p>The [L] at which half of the available ligand-binding sites are occupied (at <span class="math inline">\(\theta\)</span> = 0.5) corresponds to <span class="math inline">\(\frac{1}{K_a}\)</span>.</p>
</div><div class="column" style="width:40%;">
<p><img data-src="images/theta_ligand.png" width="400"></p>
</div></div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="dissociation-constant">Dissociation constant</h3>
<p><span class="math inline">\(K_d\)</span> is the equilibrium constant for the release of ligand.</p>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
K_d = \frac{[P][L]}{[PL]} \\
[PL] = \frac{[P][L]}{K_d} \\
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\theta = \frac{[L]}{[L] + K_d}
\end{align}\]</span> When <span class="math inline">\([L]\)</span> is equal to <span class="math inline">\(K_d\)</span>, half of the ligand-binding sites are occupied.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>As <span class="math inline">\([L]\)</span> falls below <span class="math inline">\(K_d\)</span>, progressively less of the protein has ligand bound to it.</p>
<p>In order for 90% of the available ligand-binding sites to be occupied, <span class="math inline">\([L]\)</span> must be nine times greater than <span class="math inline">\(K_d\)</span>.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="dissociation-constant-1">Dissociation constant</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math inline">\(K_d\)</span> is used much more often than <span class="math inline">\(K_a\)</span> to express the affinity of a protein for a ligand.</p>
<p>A lower value of <span class="math inline">\(K_d\)</span> corresponds to a higher affinity of the ligand for the protein.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math inline">\(K_d\)</span> is equivalent to the molar concentration of ligand at which half of the available ligand-binding sites are occupied.</p>
<p>At this point, the protein is said to have reached half-saturation with respect to ligand binding.</p>
</div>
</div>
<div class="fragment fade-in">
<p>The more tightly a protein binds a ligand, the lower the concentration of ligand required for half the binding sites to be occupied, and thus the lower the value of <span class="math inline">\(K_d\)</span>.</p>
</div>
</section>
<section class="slide level2">

<h3 id="protein-structure-affects-ligand-binding">Protein structure affects ligand binding</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>The interaction is greatly affected by protein structure and is often accompanied by conformational changes.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Carbon monoxide binds to free heme molecules more than 20,000 times better than does O<sub>2</sub> (that is, the <span class="math inline">\(K_d\)</span> or P<sub>50</sub> for CO binding to free heme is more than 20,000 times lower than that for O<sub>2</sub>).</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:60%;">
<p>CO binds only about 200 times better when the heme is bound in myoglobin.</p>
<div class="fragment fade-in">
<p>The <em>distal His</em> (His<sup>64</sup> E7), does not affect the binding of O<sub>2</sub> but may preclude the linear binding of CO.</p>
</div>
<div class="fragment fade-in">
<p>A reduction in CO binding is physiologically important, because CO is a low-level byproduct of cellular metabolism.</p>
</div>
</div><div class="column" style="width:40%;">
<p><img data-src="images/Heme_O2_CO.png" width="325"></p>
</div></div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="enzymes---introduction">Enzymes - Introduction</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Much of the history of biochemistry is the history of enzyme research.</p>
<p>Biological catalysis was first recognized and described in the late 1700s,</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>In 1897, Eduard Buchner discovered that yeast extracts could ferment sugar to alcohol, proving that fermentation was promoted by molecules that continued to function when removed from cells.</p>
</div>
</div>
<aside class="notes">
<p>German physiologist FWK - ancient greek.</p>
<p>As vitalistic notions of life were disproved, the isolation of new enzymes and the investigation of their properties advanced the science of biochemistry.</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Frederick W. Kühne called these molecules <strong>enzymes</strong>.</p>
<p>ἐν (en, “in”) +‎ ζύμη (zúmē, “sourdough”)</p>
</div>
</div>
<div class="fragment fade-in">
<div style="margin-top:1em; background-color: lightblue; padding:0.5em">
<p>A substance produced by a living organism which acts as a catalyst to bring about a specific biochemical reaction.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="most-enzymes-are-proteins">Most enzymes are proteins</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>James Sumner found that urease crystals consisted entirely of protein, and he postulated that all enzymes are proteins.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>In 1930, J. B. S. Haldane made the remarkable suggestion that weak bonding interactions between an enzyme and its substrate might be used to catalyze a reaction.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>With the exception of a small group of catalytic RNA molecules, all enzymes are proteins.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Their catalytic activity depends on the integrity of their native protein conformation.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:70%;">
<div style="margin-right:-0.8em;">
<p>Some enzymes require no chemical groups for activity other than their amino acid residues.</p>
<div class="fragment fade-in">
<p>An additional chemical component called a <strong>cofactor</strong>—either one or more inorganic ions or a complex organic or metalloorganic molecule called a <strong>coenzyme</strong>.</p>
</div>
</div>
</div><div class="column" style="width:30%;">
<p><img data-src="images/cofactors.png" width="325"></p>
</div></div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="cofactors-and-coenzymes">Cofactors and coenzymes</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Some enzymes require both a coenzyme and a cofactor for activity.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>A coenzyme or metal ion that is very tightly or even covalently bound to the enzyme protein is called a <strong>prosthetic group</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>A complete, catalytically active enzyme together with its bound coenzyme and/or metal ions is called a <strong>holoenzyme</strong>.</p>
<p>The protein part of such an enzyme is called the <strong>apoenzyme</strong> or <strong>apoprotein</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Coenzymes act as transient carriers of specific functional groups.</p>
<p>Most are derived from vitamins, organic nutrients required in small amounts in the diet.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Some enzymes are modified covalently by phosphorylation, glycosylation, and other processes.</p>
<p>Many of these alterations are involved in the regulation of enzyme activity.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="enzyme-classification">Enzyme Classification</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Many enzymes have been named by adding the suffix “-ase” to the name of their substrate or to a word or phrase describing their activity.</p>
</div>
</div>
<aside class="notes">
<p>Thus urease catalyzes hydrolysis of urea, and DNA polymerase catalyzes the polymerization of nucleotides to form DNA.</p>
<p>An enzyme known to act in the digestion of foods was named pepsin, from the Greek pepsis, “digestion,” and lysozyme was named for its ability to lyse bacterial cell walls.</p>
<p>Transfer of a phosphoryl group from ATP to glucose.</p>
<p>Number 2 denote class name (transferase); the second number (7), the subclass (phosphotransferase); the third number (1), a phosphotransferase with a hydroxyl group as acceptor; and the fourth number (1), D-glucose as the phosphoryl group acceptor. For many enzymes, a trivial name is more commonly used—in this case hexokinase.</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><img data-src="images/Enzyme_class.png"></p>
<p>Each enzyme is assigned a four-part classification number and a systematic name, which identifies the reaction it catalyzes.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\text{ATP} + \text{D-glucose} \rightarrow \text{ADP} + \text{D-glucose-6-phosphate}
\end{align}\]</span></p>
<p>Enzyme name: <strong>ATP:glucose phosphotransferase</strong> &nbsp;&nbsp;&nbsp;&nbsp; E.C. number <strong>2.7.1.1</strong></p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="how-enzymes-work">How enzymes work</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Under biologically relevant conditions, uncatalyzed reactions tend to be slow—most biological molecules are quite stable in the neutral-pH, mild temperature, aqueous environment inside cells.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Many common reactions in biochemistry entail chemical events that are unfavorable or unlikely in the cellular environment, such as the transient formation of unstable charged intermediates or the collision of two or more molecules in the precise orientation required for reaction.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>An enzyme circumvents these problems by providing a specific environment within which a given reaction can occur more rapidly.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>The distinguishing feature of an enzyme-catalyzed reaction is that it takes place within the confines of a pocket on the enzyme called the <strong>active site</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Often, the active site encloses a <strong>substrate</strong>, sequestering it completely from solution.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="enzyme-action">Enzyme action</h3>
<div style="font-size: 0.8em; margin-top:-1em;">
<p><span class="math display">\[\begin{align}
E + S \rightleftharpoons ES \rightleftharpoons EP \rightleftharpoons E + P
\end{align}\]</span></p>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em; margin-top:-0.5em;">
<p>The function of a catalyst is to increase the <strong>rate</strong> of a reaction. Catalysts do not affect reaction <strong>equilibria</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em; margin-top:-0.5em;">
<p>The free energy of the ground state of P is lower than that of S, so <span class="math inline">\(\Delta\text{G}^{′°}\)</span> for the reaction is negative and the equilibrium favors P.</p>
<p>A favorable equilibrium does not mean that the S <span class="math inline">\(\rightarrow\)</span> P conversion will occur at a detectable rate (energy “hill” ).</p>
</div>
</div><div class="column" style="width:40%;">
<p><img data-src="images/Enzyme_RS1.png"></p>
</div></div>
</div>
<div class="fragment fade-in">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em; margin-top:-0.5em;">
<p>The difference between the energy levels of the ground state and the transition state is the <strong>activation energy, <span class="math inline">\(\Delta\text{G}\)</span><sup>‡</sup></strong>.</p>
<p>Catalysts enhance reaction rates by lowering activation energies.</p>
</div>
</div><div class="column" style="width:40%;">
<div style="margin-top:-1em;">
<p><img data-src="images/Enzyme_RS2.png"></p>
</div>
</div></div>
</div>
</section>
<section class="slide level2">

<h5 id="substrate-concentration-affects-the-rate-of-enzyme-catalyzed-reactions">Substrate Concentration Affects the Rate of Enzyme-Catalyzed Reactions</h5>
<div class="fragment">
<div class="columns">
<div class="column" style="width:40%;">
<p><img data-src="images/Vmax.png"></p>
</div><div class="column" style="width:60%;">
<div style="font-size: 0.8em; margin-top:-1em;">
<p>Initial rate (<strong>V<sub>0</sub></strong>), when [S] &gt;&gt; [E].</p>
<p><span class="math display">\[\begin{align}
E + S \rightleftharpoons ES \;\;\;\;\;\; \text{(fast)}\\
ES \rightleftharpoons E + P \;\;\;\;\;\; \text{(slow)}
\end{align}\]</span></p>
<div class="fragment fade-in">
<p>The reaction quickly achieves a steady state in which [ES] remains approximately constant over time (<strong>steady-state kinetics</strong>).</p>
</div>
</div>
</div></div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
E + S \underset{k_{-1}}{\stackrel{k_1}{\rightleftharpoons}} ES \underset{}{\stackrel{k2}{\rightarrow}} E + P
\end{align}\]</span></p>
</div>
</div>
<aside class="notes">
<p>Assumption, P to S is ignored i.e.&nbsp;no k-2.</p>
<p>V0 is determined by the breakdown of ES to form product, which is determined by [ES]</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
V_o = k_2 [ES]
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>[ES] can not be easily measured experimentally.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="enzyme-kinetics">Enzyme kinetics</h3>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\text{Rate of ES formation = } k_1 ([E_t] - [ES])[S] \\
\text{Rate of ES breakdown = } k_{-1} [ES] + k_2 [ES]
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><strong>Steady-state assumption:</strong> Rate of formation of ES is equal to the rate of its breakdown. <span class="math display">\[\begin{align}
k_1 ([E_t] - [ES])[S] =  k_{-1} [ES] + k_2 [ES]
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
[ES] =  \frac{[E_t][S]}{[S] + (k_2 + k_{-1})/k_1}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
[ES] =  \frac{[E_t][S]}{K_m + [S]}
\end{align}\]</span> <span class="math inline">\(K_m\)</span> is <strong>Michaelis constant</strong>.</p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="michaelis-menten-equation">Michaelis-Menten Equation</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Expressing <span class="math inline">\(V_0\)</span> in terms of <span class="math inline">\([ES]\)</span>. <span class="math display">\[\begin{align}
V_0 =  \frac{k_2 [E_t][S]}{K_m + [S]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Maximum velocity occurs when the enzyme is saturated (i.e.&nbsp;<span class="math inline">\([ES] = [E_t]\)</span>).</p>
<div style="font-size: 1em;">
<p><span class="math display">\[\begin{align}
V_0 =  \frac{V_{max} [S]}{K_m + [S]}
\end{align}\]</span></p>
</div>
</div>
</div>
<aside class="notes">
<p>This is the Michaelis-Menten equation, the rate equation for a one-substrate enzyme-catalyzed reaction. It is a statement of the quantitative relationship between the initial velocity V0, the maximum velocity Vmax, and the initial substrate concentration [S], all related through the Michaelis constant Km. Note that Km has units of concentration. Does the equation fit experimental observations? Yes; we can confirm this by considering the limiting situations where [S] is very high or very low, as shown in Figure 6–12.</p>
<p>This is a very useful, practical definition of Km: Km is equivalent to the substrate concentration at which V0 is one-half Vmax.</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>When <span class="math inline">\(V_o\)</span> is half <span class="math inline">\(V_{max}\)</span> <span class="math display">\[\begin{align}
\frac{V_{max}}{2} =  \frac{V_{max} [S]}{K_m + [S]} =&gt; \frac{1}{2} = \frac{[S]}{K_m + [S]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Solving of <span class="math inline">\(K_m\)</span>, we get <span class="math inline">\(K_m + [S] = 2[S]\)</span> or <span class="math inline">\(K_m = [S]\)</span>, when <span class="math inline">\(V_0 = \frac{1}{2}V_{max}\)</span></p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="double-reciprocal-plot">Double-Reciprocal Plot</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
V_0 =  \frac{V_{max} [S]}{K_m + [S]} \text{ Reciprocal =&gt; }  \frac{1}{V_0} = \frac{K_m + [S]}{V_{max}[S]}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><span class="math display">\[\begin{align}
\frac{1}{V_0} = \frac{K_m}{V_{max}[S]} + \frac{[S]}{V_{max}[S]} =&gt;
\frac{1}{V_0} = \frac{K_m}{V_{max}[S]} + \frac{1}{V_{max}}
\end{align}\]</span></p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em;">
<p>A double-reciprocal plot or <strong>Lineweaver-Burk plot</strong>.</p>
<div class="fragment fade-in">
<p>Slope =&gt; <span class="math inline">\(\frac{K_m}{V_{max}}\)</span> Y intercept =&gt; <span class="math inline">\(\frac{1}{V_{max}}\)</span> &nbsp;&nbsp; X intercept =&gt; <span class="math inline">\(-\frac{1}{K_m}\)</span></p>
</div>
<div class="fragment fade-in">
<p>The double-reciprocal plot of enzyme reaction rates is very useful in distinguishing between certain types of enzymatic reaction mechanisms and in analyzing enzyme inhibition.</p>
</div>
</div>
</div><div class="column" style="width:40%;">
<div style="font-size: 0.8em;">
<p><img data-src="images/Double_reciprocal.png" width="400"></p>
</div>
</div></div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="turnover-number">Turnover number</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Number of substrate molecules converted to product in a given unit of time on a single enzyme molecule when the enzyme is saturated with substrate.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Each enzyme has values of <span class="math inline">\(k_{cat}\)</span> and <span class="math inline">\(K_m\)</span> that reflect the cellular environment, the concentration of substrate normally encountered in vivo by the enzyme, and the chemistry of the reaction being catalyzed.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Two enzymes catalyzing different reactions may have the same <span class="math inline">\(k_{cat}\)</span> (turnover number), yet the rates of the uncatalyzed reactions may be different and thus the rate enhancements brought about by the enzymes may differ greatly.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>The best way to compare the catalytic efficiencies of different enzymes or the turnover of different substrates by the same enzyme is to compare the ratio <span class="math inline">\(\frac{k_{cat}}{K_m}\)</span> for the two reactions.</p>
</div>
</div>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><span class="math inline">\(\frac{k_{cat}}{K_m}\)</span> is called the <strong>specificity constant</strong>.</p>
</div>
</div>
<aside class="notes">
<p>This parameter, sometimes called the specificity constant, is the rate constant for the conversion of E+S to E+P. When [S] &lt;&lt; Km.</p>
<p>V0 in this case depends on the concentration of two reactants, [Et] and [S]; therefore this is a second-order rate equation and the constant.</p>
<p>There is an upper limit to kcat/Km, imposed by the rate at which E and S can diffuse together in an aqueous solution. This diffusion controlled limit is 10e8 to 10e9 M-1 s-1</p>
<p>Many enzymes have a kcat/Km near this range (Table 6–8). Such enzymes are said to have achieved catalytic perfection. Note that different values of kcat and Km can produce the maximum ratio</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section class="slide level2">

<h3 id="presteady-state-kinetics">Pre–Steady State Kinetics</h3>
<div class="fragment">
<div class="columns">
<div class="column" style="width:60%;">
<div style="font-size: 0.8em;">
<p>Sequential vs double-displacement (ping-pong) reactions.</p>
<div class="fragment fade-in">
<p>In Sequential kinetics there is formation of a ternary complex.</p>
</div>
<div class="fragment fade-in">
<p>In ping-pong kinetics one of the products is release before all the substrates have bound to the enzyme.</p>
</div>
<div class="fragment fade-in">
<div style="margin-top: -1em;">
<p><img data-src="images/ping_pong.png" width="500"></p>
</div>
</div>
</div>
</div><div class="column" style="width:40%;">
<div style="font-size: 0.8em;">
<p><img data-src="images/Double_reciprocal_2.png" width="400"></p>
</div>
</div></div>
</div>
<aside class="notes">
<p>The sequential mechanism can be ordered or random (for substrate binding).</p>
<p>Kinetic Tests for Determining Inhibition Mechanisms</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section class="slide level2">

<h3 class="smaller" id="regulatory-enzymes">Regulatory Enzymes</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p>Each pathway, however, includes one or more enzymes that have a greater effect on the rate of the overall sequence. These regulatory enzymes exhibit increased or decreased catalytic activity in response to certain signals.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Mostly, the first enzyme of the sequence is a regulatory enzyme.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p><strong>Allosteric enzymes</strong> function through reversible, noncovalent binding of regulatory compounds called allosteric modulators or allosteric effectors (small metabolites or cofactors).</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Other enzymes are regulated by reversible <strong>covalent modification</strong>.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Both classes of regulatory enzymes tend to be multisubunit proteins, and in some cases the regulatory site(s) and the active site are on separate subunits.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Some enzymes are stimulated or inhibited when they are bound by separate regulatory proteins.</p>
</div>
</div>
<div class="fragment fade-in">
<div style="font-size: 0.8em;">
<p>Others are activated when peptide segments are removed by proteolytic cleavage.</p>
</div>
</div>
<aside class="notes">
<p>Examples of enzymes activated by proteolytic cleavage? What is feedback inhibition?</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section class="slide level2">

<h3 id="allosteric-enzymes">Allosteric enzymes</h3>
<div class="fragment">
<div style="font-size: 0.8em;">
<p><img data-src="images/Allostery.png"></p>
</div>
</div>
<aside class="notes">
<p>The Kinetic Properties of Allosteric Enzymes Diverge from Michaelis-Menten Behavior</p>
<p>Examples of allosteric enzymes? Aspartate transcarbomylase</p>
<p>Enzymes at important metabolic intersections may be regulated by complex combinations of effectors, allowing coordination of the activities of interconnected pathways.</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>


</section>
    </div>
  <div class="quarto-auto-generated-content" style="display: none;">
<p><img src="images/MITWPU_logo.jpg" class="slide-logo"></p>
<div class="footer footer-default">

</div>
</div></div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'jumpToSlide': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleScrollView(event)\"><kbd>r</kbd> Scroll View Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'speaker',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    <script id="quarto-html-after-body" type="application/javascript">
      window.document.addEventListener("DOMContentLoaded", function (event) {
        const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
        tabsets.forEach(function(tabset) {
          const tabby = new Tabby('#' + tabset.id);
        });
        const isCodeAnnotation = (el) => {
          for (const clz of el.classList) {
            if (clz.startsWith('code-annotation-')) {                     
              return true;
            }
          }
          return false;
        }
        const onCopySuccess = function(e) {
          // button target
          const button = e.trigger;
          // don't keep focus
          button.blur();
          // flash "checked"
          button.classList.add('code-copy-button-checked');
          var currentTitle = button.getAttribute("title");
          button.setAttribute("title", "Copied!");
          let tooltip;
          if (window.bootstrap) {
            button.setAttribute("data-bs-toggle", "tooltip");
            button.setAttribute("data-bs-placement", "left");
            button.setAttribute("data-bs-title", "Copied!");
            tooltip = new bootstrap.Tooltip(button, 
              { trigger: "manual", 
                customClass: "code-copy-button-tooltip",
                offset: [0, -8]});
            tooltip.show();    
          }
          setTimeout(function() {
            if (tooltip) {
              tooltip.hide();
              button.removeAttribute("data-bs-title");
              button.removeAttribute("data-bs-toggle");
              button.removeAttribute("data-bs-placement");
            }
            button.setAttribute("title", currentTitle);
            button.classList.remove('code-copy-button-checked');
          }, 1000);
          // clear code selection
          e.clearSelection();
        }
        const getTextToCopy = function(trigger) {
            const codeEl = trigger.previousElementSibling.cloneNode(true);
            for (const childEl of codeEl.children) {
              if (isCodeAnnotation(childEl)) {
                childEl.remove();
              }
            }
            return codeEl.innerText;
        }
        const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
          text: getTextToCopy
        });
        clipboard.on('success', onCopySuccess);
        if (window.document.getElementById('quarto-embedded-source-code-modal')) {
          const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
            text: getTextToCopy,
            container: window.document.getElementById('quarto-embedded-source-code-modal')
          });
          clipboardModal.on('success', onCopySuccess);
        }
          var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
          var mailtoRegex = new RegExp(/^mailto:/);
            var filterRegex = new RegExp('/' + window.location.host + '/');
          var isInternal = (href) => {
              return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
          }
          // Inspect non-navigation links and adorn them if external
         var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
          for (var i=0; i<links.length; i++) {
            const link = links[i];
            if (!isInternal(link.href)) {
              // undo the damage that might have been done by quarto-nav.js in the case of
              // links that we want to consider external
              if (link.dataset.originalHref !== undefined) {
                link.href = link.dataset.originalHref;
              }
            }
          }
        function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
          const config = {
            allowHTML: true,
            maxWidth: 500,
            delay: 100,
            arrow: false,
            appendTo: function(el) {
                return el.closest('section.slide') || el.parentElement;
            },
            interactive: true,
            interactiveBorder: 10,
            theme: 'light-border',
            placement: 'bottom-start',
          };
          if (contentFn) {
            config.content = contentFn;
          }
          if (onTriggerFn) {
            config.onTrigger = onTriggerFn;
          }
          if (onUntriggerFn) {
            config.onUntrigger = onUntriggerFn;
          }
            config['offset'] = [0,0];
            config['maxWidth'] = 700;
          window.tippy(el, config); 
        }
        const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
        for (var i=0; i<noterefs.length; i++) {
          const ref = noterefs[i];
          tippyHover(ref, function() {
            // use id or data attribute instead here
            let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
            try { href = new URL(href).hash; } catch {}
            const id = href.replace(/^#\/?/, "");
            const note = window.document.getElementById(id);
            if (note) {
              return note.innerHTML;
            } else {
              return "";
            }
          });
        }
        const findCites = (el) => {
          const parentEl = el.parentElement;
          if (parentEl) {
            const cites = parentEl.dataset.cites;
            if (cites) {
              return {
                el,
                cites: cites.split(' ')
              };
            } else {
              return findCites(el.parentElement)
            }
          } else {
            return undefined;
          }
        };
        var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
        for (var i=0; i<bibliorefs.length; i++) {
          const ref = bibliorefs[i];
          const citeInfo = findCites(ref);
          if (citeInfo) {
            tippyHover(citeInfo.el, function() {
              var popup = window.document.createElement('div');
              citeInfo.cites.forEach(function(cite) {
                var citeDiv = window.document.createElement('div');
                citeDiv.classList.add('hanging-indent');
                citeDiv.classList.add('csl-entry');
                var biblioDiv = window.document.getElementById('ref-' + cite);
                if (biblioDiv) {
                  citeDiv.innerHTML = biblioDiv.innerHTML;
                }
                popup.appendChild(citeDiv);
              });
              return popup.innerHTML;
            });
          }
        }
      });
      </script>
    

</body></html>